libs/protocol-client/src/lib/client/contribution.ts

```diff
@@ -164,6 +170,14 @@ export class Contribution extends BaseClient {
   ) {
     const contract = new GovrnContract(networkConfig, signer);
     const transaction = await contract.mint(args);
+    await this.sdk.updateUserOnChainContribution({
+      id: id,
+      status: 'pending',
```

Comment:
I looked for the frontend to check for technically affected code; there are none. 

But I have some observations:
- Shouldn't editing be [disabled](https://github.com/Govrn-HQ/govrn-monorepo/blob/7925c9f35c097fe03c520a6e9803780342464f6a/apps/protocol-frontend/src/components/ContributionsTable.tsx#L205) for `pendig` status too? if editing is enabled, could this duplicate contributions for contract sync job?
- We might need to look for a third [emoji](https://github.com/Govrn-HQ/govrn-monorepo/blob/a724a9c967485dedbac57b247e15655cfc21ab53/apps/protocol-frontend/src/components/ContributionDetailShell.tsx#L71-L73) for `pending`!

Reply:
Good call, yes we should disable and we can add another emoji